src/stories/components/meter/Meter.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta, Title, Canvas } from '@storybook/blocks';
+import CustomArgTypes from '../../blocks/CustomArgTypes';
+import * as MeterStories from './Meter.stories';
+
+<Meta of={MeterStories} />
+
+<Title />
+
+Meters are used to represent scalar values within a known range.
+
+<Canvas of={MeterStories.Demo} />
+
+### Tickmarks
+
+Meters can be configured to display tickmarks at regular intervals along the track. The `tickmarks`
+property toggles whether or not tickmarks are present and the number of tickmarks is set via the
+`--forge-meter-tickmarks` token.
+
+<Canvas of={MeterStories.Tickmarks} />
+
+### Segmented
+
+Setting a meter's `low` or `high` value turns it into a segemented meter. A segmented meter's is
+divided into three regions — below the low value, above the high value, and between the two. Its
+color changes based on the value of the meter in relation to its `optimum` property. When the value
+is in the same region as the optimum, the value is _optimal_ and colored green. When it's in an
+adjecent region it's _suboptimal_ and colored yellow. Otherwise, its _least optimal_ and colored
+red.
+
+> When set, `low` must be greater than or equal to `min` and less than `high`. Likewise, `high`
+> must be less than or equal to `max` and greater than `low`.
+
+<Canvas of={MeterStories.Segmented} />
+
+### Grouped
+
+Multiple related meters representing parts of a whole can be grouped together to display on the
+same track. Each meter in the group must have the same min and max values, which are managed by
+the `<forge-meter-group>` element, and can't be segmented.
+
+Grouped meters are typically accompanied by keys that label each meter with its color and
+optionally its value.
+
+<Canvas of={MeterStories.Grouped} />
+
```

Comment:
May be worth adding some supporting content about the usage of the new key component here, and a link to the key docs page.